build_scripts/build_test_dependencies.txt

```diff
@@ -0,0 +1,16 @@
+pyspelling==2.6
+mutmut==1.5.1
+pytest==5.0.0
+xlrd==1.2.0
+xlsxwriter==1.2.1
+codecov==2.0.15
+pytest-cov==2.7.1
+pylint==2.3.1
+pandas==1.0.5
+scikit-learn==0.23.1
+numpy==1.18.5
```

Comment:
i am not sure where these dependencies are used ?

Reply:
corrected